job/job.go

```diff
@@ -94,6 +126,11 @@ type Metadata struct {
 	LastAttemptedRun time.Time `json:"last_attempted_run"`
 }
 
+type Header struct {
```

Comment:
the difference is in how the user request will look like

currently:

`{"headers": [{"Content-Type": "text/plain"}]`

vs using Google's:

`{"headers": {"Content-Type": ["text/plain"]}}`

the first seemed a little more natural